src/conversion_helper.js

```diff
@@ -429,7 +429,7 @@ const unitLookupList = [
     "imperialUnits" : [/cups?/],
     "standardInputUnit" : " cups (US)",
     "isInvalidInput" : isZeroOrNegative,
-    "isWeaklyInvalidInput" : isHyperbole,
+    "isWeaklyInvalidInput" : (i) => i > 100,
```

Comment:
We'd still like to check for `isHyperbole` as well, please make this an or condition here.

Reply:
I added it back in.